6238_prototype/src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.SmartDashboardParam;
+import frc.robot.subsystems.BallSubsystem;
+
+public class ShooterCommand extends CommandBase{
+    private final BallSubsystem ball; 
+    private final SmartDashboardParam upperShooterSpeedTarget;
+    public ShooterCommand(BallSubsystem ball) {
+        this.ball = ball;
+        addRequirements(ball);
+        upperShooterSpeedTarget = new SmartDashboardParam("upperMotorSpeedTarget");
+    }
+
+    public void execute() {
+        ball.extend();
+        double upperMotorSpeedTarget = upperShooterSpeedTarget.get();
+        double upperMotorSpeedCurrent = ball.getSpeedUpperMotor();
+        SmartDashboard.putNumber("upperMotorSpeedCurrent", upperMotorSpeedCurrent);
```

Comment:
You never call `ball.setSpeed`.  You need to do that from this function so that the shooter motor starts running.